big_tests/tests/connect_SUITE.erl

```diff
@@ -46,9 +46,9 @@ all() ->
     ].
 
 groups() ->
-    G = [ {c2s_noproc, [], [reset_stream_noproc,
-                            starttls_noproc,
-                            compress_noproc,
+    G = [ {c2s_suspend, [], [reset_stream_suspend,
+                            starttls_suspend,
```

Comment:
The indentation here look broken (at least on github). In most of other places the second element in the list (in this case `starttls_suspend`) starts in the same column as the first element above.